mcstatus/utils.py

```diff
@@ -131,6 +132,7 @@ def deprecated(
     msg: str | None = None,
     methods: Iterable[str] | None = None,
 ) -> Callable | type[object]:
+    # TODO add ``.. deprecated::`` to docstring
```

Comment:
Same remarks as with my other comment regarding TODOs in code.

Reply:
Created #460, removed [here](https://github.com/py-mine/mcstatus/compare/b4c7e9763c5d6f7138586c81b52947c1894b9073..bb194c20f1882cd4832076a1360070ec9bffce64).